common/test/acceptance/tests/studio/test_studio_course_info.py

```diff
@@ -0,0 +1,145 @@
+"""
+Acceptance Tests for Course Information
+"""
+from common.test.acceptance.pages.studio.course_info import CourseUpdatesPage
+from common.test.acceptance.tests.studio.base_studio_test import StudioCourseTest
+
+from ...pages.studio.auto_auth import AutoAuthPage
+from ...pages.studio.index import DashboardPage
+
+
+class UsersCanAddUpdatesTest(StudioCourseTest):
+    """
+    Series of Bok Choy Tests to test the Course Updates page
+    """
+
+    def _create_and_verify_update(self, message):
+        self.course_updates_page.visit()
+        self.assertTrue(self.course_updates_page.is_new_update_button_present())
+        self.course_updates_page.click_new_update_button()
+        self.course_updates_page.submit_update(message)
+        self.assertTrue(self.course_updates_page.is_first_update_message(message))
+
+    def setUp(self, is_staff=False, test_xss=True):
+        super(UsersCanAddUpdatesTest, self).setUp()
+        self.auth_page = AutoAuthPage(self.browser, staff=True)
+        self.dashboard_page = DashboardPage(self.browser)
+        self.course_updates_page = CourseUpdatesPage(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+
+    def test_course_updates_page_exists(self):
+        """
+        Scenario: User can access Course Updates Page
+            Given I have opened a new course in Studio
+            And I go to the course updates page
+            When I visit the page
+            Then I should see any course updates
+            And I should see the new update button
+        """
+        self.course_updates_page.visit()
+        self.course_updates_page.wait_for_page()
+        self.assertTrue(self.course_updates_page.are_course_updates_on_page())
```

Comment:
Should course updates always be visible on the page, or just in this one case? I noticed you only check for it here.

If it's always supposed to be there, you could put this in the is_browser_on_page method.